manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
+    ensure => 'directory',
+    owner  => 'root',
+    group  => '0',
+    mode   => '0755',
+  }
```

Comment:
Are you worried about it reaching this resource before $mcollective::ssldir is evaluated? I thought that's what the 'contain' function was supposed to prevent?